src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/CaseDataStoreService.java

```diff
@@ -14,5 +14,5 @@
      * @param documentId Document Id
      * @return Optional containing Case Metadata when found; empty optional otherwise
      */
-    Optional<CaseDocumentMetadata> getCaseDocumentMetadata(final String caseReference, UUID documentId);
+    Optional<Document> getCaseDocumentMetadata(final String caseReference, UUID documentId);
```

Comment:
Would it be better to rename this "Document" entity to "DocumentPermission" as it contains only permission for a particular document?

Reply:
yes, I think, that would be better.

Reply:
updated to DocumentPermissions.